src/content/documentation/develop/debugging.md

```diff
@@ -90,6 +92,41 @@ You can now drag the toolbox tab to a separate window, so you can place it along
 
 {% capture content %}
 
+## Viewing log output
+
+Your extension can generate log messages, e.g. with the [Console API](https://developer.mozilla.org/docs/Web/API/Console_API) or by triggering (uncaught) errors.
+
+Log messages can usually be viewed via the [Console](https://developer.mozilla.org/docs/Tools/Web_Console) of the context where the script runs.
+The [Developer tools toolbox](#developer-tools-toolbox) at `about:debugging` shows log messages from extension scripts. For specific cases, see:
+
+  - [Debugging background scripts](#debugging-background-scripts)
+  - [Debugging options pages](#debugging-options-pages)
+  - [Debugging popups](#debugging-popups)
+  - [Debugging sidebars](#debugging-sidebars)
+  - [Debugging devtools panels](#debugging-developer-tools-pages-and-panels).
+
+Log messages from content scripts can be viewed in the developer tools of the tab where the content script runs instead of `about:debugging`, see:
+
+- [Debugging content scripts](#debugging-content-scripts)
+
```

Comment:
(even just as a follow up) here or in that "Debugging content scripts" section we should add a warning box to mention explicitly [Bug 1410932](https://bugzilla.mozilla.org/show_bug.cgi?id=1410932) and to look for those errors in the Browser Console (after enabling the "Content Messages" toggle button). 

It may be better to do it in a separate follow up, we may want to double-check explicitly which are the cases that are still not logged in the webconsole, evilpie was the last one to give a try to that issue but from what I recall we were not able to get a reasonable solution that would cover all cases properly, but based on comment 43 it seems that [bug 1696756](https://bugzilla.mozilla.org/show_bug.cgi?id=1696756) may have covered the rejected promises.